VegProcessor/species_hsi/blhwva.py

```diff
@@ -73,13 +73,28 @@ def __post_init__(self):
         # Calculate overall suitability score with quality control
         self.hsi = self.calculate_overall_suitability()
 
-    def _create_template_array(self) -> np.ndarray:
-        """Create an array from a template all valid pixels are 999.0, and
-        NaN from the input are persisted.
+    def _create_template_array(self, *input_arrays) -> np.ndarray:
```

Comment:
Adding an argument so that additional arrays can be passed, specifically to propagate their NaNs into the template.